guides/Getting-Started.md

```diff
@@ -0,0 +1,77 @@
+# Getting Started With Test262
+
+
+Rough outline...
+
+
+- Why do we test?
+- What do we test? 
+  - Similar to https://github.com/tc39/test262/blob/main/CONTRIBUTING.md#acceptable-tests and https://github.com/tc39/test262/blob/main/CONTRIBUTING.md#unacceptable-tests
+- Anatomy of a test
+  - frontmatter
+    - esid
+    - description
+    - don't mention any other frontmatter
+  - test code
+    - assert api
+- Let's write a test, end to end.
+  - Pre-work
+    - Find something that needs to be tested
+    - Read spec
+    - Create test plan
+      - This can be a super minimal version of the usual test plans, but will help to show how we work
+        - Write a checklist (we'll give a 2-3 point example to work from)
+      - Does it need a new feature tag?
+    - Share test plan
+  - Writing our first test
+    - Git stuff
+      - Pull test272 repo
+      - Create a branch
+      - What else?
+    - Create a new file
+      - Where? Why?
```

Comment:
As far as I can tell, unless I missed something somewhere, there are only three places where that `Foo/constructor/` path pattern occurred: 

- test/intl402/ListFormat/constructor
- test/intl402/RelativeTimeFormat/constructor
- test/intl402/Segmenter/constructor

...I've had a "TODO" in some eng. journal of mine (somewhere) to fix them. I think I merged them and wrote myself the note and never followed up. The reason there isn't a `built-ins/X/constructor` is because `X.constructor` is `Function` and those tests are in `built-ins/Function`. One might argue that constructor-specific tests maybe belong in `built-ins/X/prototype/constructor`, however I would counter with: `X.prototype.constructor === X`, so `built-ins/X` is semantically the same and much less to type.

Reply:
Yeah, I didn't mean I was super bothered by the discrepancy of `constructor/`, but maybe we could solve the larger question by having a "Where to put my test?" guide document. I could try to throw an outline together during this work session time.